app/views/provider/admin/api_docs/show.html.erb

```diff
@@ -1,9 +1,8 @@
-<%= stylesheet_link_tag 'active-docs/application.css' %>
 <%= javascript_tag do %>
   window.enableApiDocsProxy = <%= Rails.configuration.three_scale.active_docs_proxy_disabled.blank? %>;
 <% end %>
+<%= javascript_pack_tag 'provider_active_docs' %>
 <%= javascript_include_tag 'active-docs/application.js' %>
```

Comment:
Having these 2 together...

![](https://media1.giphy.com/media/3owzW7ZwPoIbxHkpgc/giphy.gif?cid=ecf05e47f4hty7adtm2grsb1lho4m9mlk26kfu72pevmsxid&rid=giphy.gif&ct=g)

It reeks of technical debt. I see most of the imports are scripts injected into the global scope (💩), also old jquery https://github.com/3scale/porta/blob/25fa9bc53bdd3eeb8f2a9c8e882f771c85c3c3cb/vendor/active-docs/vendor/assets/javascripts/active-docs/vendor.js#L1 
But who knows what's old and what's still used. Maybe some day we can clean it up.

Reply:
Good catch. Actually, I don't know whether we need that here, most likely we don't.

Reply:
Hm, I thought I commented on this, but don't see the comment now.
I think we might not need this here, actually, so I'll remove for now.